hack/testing/utils

```diff
@@ -1,410 +0,0 @@
-#!/bin/bash
-
-millisecond=1
-second=$(( 1000 * millisecond ))
-minute=$(( 60 * second ))
-
-LOGGING_NS="openshift-logging"
-
-log::info(){
-  echo " [INFO] $@"
-}
-
-log::error(){
-  echo "[ERROR] $@"
-}
-
-indent() {
-  INDENT="      "
-  sed "s/^/$INDENT/" | sed "s/^${INDENT}\($1\)/${INDENT:0:-2}- \1/"
-}
-
-expect_success(){
-  local cmd=$1
-  echo "Running '$cmd'"
-  if $cmd ; then
-    return 0
-  fi  
-  return 1
-}
-
-try_until_failure() {
-  local cmd=$1
-  local timeout=$2
-  local interval=${3:-0.2}
-  local now=$(date +%s)
-  local expire=$(($now + $timeout))
-  while [ $now -lt $expire ]; do
-    if ! $cmd ; then
-      return 0
-    fi  
-    sleep $interval
-    now=$(date +%s)
-  done
-  return 1
-}
-try_until_success() {
-  local cmd=$1
-  local timeout=$2
-  local interval=${3:-0.2}
-  local now=$(date +%s%3N)
-  local expire=$(($now + $timeout))
-  while [ $now -lt $expire ]; do
-    if $cmd ; then
-      return 0
-    fi  
-    sleep $interval
-    now=$(date +%s%3N)
-  done
-  return 1
-}
-
-try_until_text() {
-  local cmd=$1
-  local expected=$2
-  local timeout=$3
-  local interval=${4:-0.2}
-  local now=$(date +%s%3N)
-  local expire=$(($now + $timeout))
-  while [ $now -lt $expire ]; do
-    if [[ "$($cmd)" == "${expected}" ]] ; then
-      return 0
-    fi  
-    sleep $interval
-    now=$(date +%s%3N)
-  done
-  log::error "try_until_text expired for '$cmd' == '$expected'"
-  return 1
-}
-
-gather_logging_resources() {
```

Comment:
The corresponding method in `hack/testing-olm/utils` is not this detailed.
Copy to `testing-olm` before deleting.

Reply:
done, thanks for catching it.